components/camel-milo/src/main/java/org/apache/camel/component/milo/server/internal/CamelServerItem.java

```diff
@@ -55,7 +55,7 @@ public CamelServerItem(final String itemId, final UaNodeContext nodeContext, fin
         this.itemId = itemId;
         this.baseNode = baseNode;
 
-        final NodeId nodeId = new NodeId(namespaceIndex, "items-" + itemId);
+        final NodeId nodeId = new NodeId(namespaceIndex, "" + itemId);
```

Comment:
I think you can remove the empty string concatenation.

Reply:
yes, of course. it is done.